linera-chain/src/test.rs

```diff
@@ -142,24 +142,42 @@ impl VoteTestExt for Vote {
 /// Helper trait to simplify constructing messages for tests.
 pub trait MessageTestExt: Sized {
     fn to_simple_incoming(self, sender: ChainId, height: BlockHeight) -> IncomingBundle;
+
+    fn to_posted(self, index: u32, kind: MessageKind) -> PostedMessage;
 }
 
 impl<T: Into<Message>> MessageTestExt for T {
     fn to_simple_incoming(self, sender: ChainId, height: BlockHeight) -> IncomingBundle {
+        let posted_message = PostedMessage {
+            authenticated_signer: None,
+            grant: Amount::ZERO,
+            refund_grant_to: None,
+            kind: MessageKind::Protected,
+            index: 0,
+            message: self.into(),
+        };
```

Comment:
Optional: I think we could use `to_posted` here:

```suggestion
        let posted_message = self.to_posted(0, MessageKind::Protected);
```

Reply:
Actually, the function has the wrong name: It's not simple, but protected! Also, that's not even appropriate in one of the call sites! I'll just remove it completely.